pkgs/development/python-modules/ansible-lint/default.nix

```diff
@@ -82,7 +91,7 @@ buildPythonPackage rec {
     "test_discover_lintables_umlaut"
   ];
 
-  makeWrapperArgs = [ "--prefix PATH : ${lib.makeBinPath [ ansible-base ]}" ];
+  makeWrapperArgs = [ "--prefix PATH : ${lib.makeBinPath [ ansible-core ]}" ];
```

Comment:
We should use a patch instead because this won't work if people import ansible-lint instead of running the executable.

Reply:
Is that something people actually do?

Maybe I just step down from ansible-lint maintenance since I haven't used it for almost a year by now.

Reply:
I don't know if people use it that way. But it is in `pythonPackages` and some package depends on it.

Reply:
I don't think any package depends on it https://gist.github.com/GrahamcOfBorg/b11b079c306b2cf85de90b3602db9981

and we should probably move it out of pythonPackages

Reply:
Oh, you're right.
According to https://ansible-lint.readthedocs.io/en/latest/pkg/modules/ the Python API is "private" and "unsupported", so moving it out of pythonPackages seems like the correct solution indeed.